XenModel/Actions/OvfActions/Import.cs

```diff
@@ -490,20 +490,20 @@ private XenRef<VM> CreateVm(string vmName, VirtualHardwareSection_Type system, X
             if (rasds != null && rasds.Length > 0)
             {
                 //The default memory unit is MB (2^20), however, the RASD may contain a different
-                //one with format Bytes*memoryBase^memoryPower (Bytes being a literal string)
+                //one with format byte*memoryBase^memoryPower (byte being a literal string)
 
                 double memoryPower = 20.0;
                 double memoryBase = 2.0;
                
                 foreach (RASD_Type rasd in rasds)
                 {
-                    if (rasd.AllocationUnits.Value.ToLower().StartsWith("bytes"))
+                    if (rasd.AllocationUnits.Value.ToLower().StartsWith("byte"))

```

Comment:
This was unrelated with the current PR, but I noticed it when I made the correction in ImportSelectHostPage. I think the ony reason why it hasbeen working so far is because in most cases the values are the same as the defaults specified above.